docs/contribute/DEV_DOCS.md

```diff
@@ -1,18 +1,6 @@
 # Augur Development Docs for Contributors
 
-Thank you for helping us to improve Augur! This document describes:
-
-- Getting Started
-- Contributing code
-  - Running local code changes
-  - Testing
-  - Releasing
-  - Maintaining Bioconda package
-  - Continuous integration
-- Contributing documentation
-  - Formats (Markdown and reStructuredText)
-  - Documentation structure
-  - Building documentation
+Thank you for helping us to improve Augur! Use the GitHub markdown preview sidebar to navigate the sections in this document.
```

Comment:
I've always missed this from our docs. The original auspice docs had this, and when it works it's amazing (it only half works on GitHub).

Reply:
The GitHub preview sidebar works fine for me, but I'm not a huge fan of relying on it.

An embedded table of contents with links to each section (and only up to a certain depth) seems beneficial to me. There's VSCode extensions that automatically insert/maintain one, but that's relying on a tool that not everyone editing this file will use, and I'm not sure those can ignore sections after a certain depth.

I'd be open to manually adding/maintaining one at the top level, something like [this](https://github.com/nextstrain/augur/blob/f0449eca01a386f2e9c8cb12883dbac376e0ea07/docs/contribute/DEV_DOCS.md). Thoughts?

Reply:
Oh, I was referring to page designs which use the right margin to display a floating TOC, and then (bonus) highlight the entry you're currently reading. For example, [here's](https://developer.redis.com/develop/python/) a page built using docusaurus, which was what Auspice initially used for its docs.

Reply:
Oh! That does look much nicer. Probably not for this file unless we host it on RTD, but it sounds like some other of our RTD pages could benefit from this feature. Created an issue: https://github.com/nextstrain/docs.nextstrain.org/issues/161